benchflow-dsl/src/main/scala/cloud/benchflow/dsl/definition/simone/BenchFlowExperiment.scala

```diff
@@ -0,0 +1,40 @@
+package cloud.benchflow.dsl.definition.simone
+
+import cloud.benchflow.dsl.definition._
+import cloud.benchflow.dsl.definition.simone.properties.Properties
+import cloud.benchflow.dsl.definition.sut.Sut
+import cloud.benchflow.dsl.definition.sut.configuration.SutConfiguration
+
+/**
+  * @author Simone D'Avico (simonedavico@gmail.com)
+  *
+  * Created on 11/02/16.
+  */
+case class Users(users: Int)
+case class BenchFlowExperiment(name: String,
+                               description: String,
+                               sut: Sut,
+                               users: Users,
+                               drivers: Seq[Driver[_ <: Operation]],
+                               trials: TotalTrials,
+                               execution: LoadFunction,
+                               properties: Properties,
+                               sutConfiguration: SutConfiguration)
+{
+  def getAliasForService(serviceName: String) = sutConfiguration.deployment.get(serviceName)
+  // TODO - move me
```

Comment:
If we need helper methods I think that is fine, however, we can get it also directly from the BenchFlowTest/Experiment instance.